api/db/changelog/db.changelog-243-replace-extension-count.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+  xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+  <changeSet author="evrii" id="db.changelog-243-replace-extension-count">
+    <dropColumn tableName="user_initial_credits_expiration" columnName="extension_count"/>
```

Comment:
Extension count is no longer needed, because we can only extend once. Extension time can serve as a stand in for extension count for checking whether someone has requested an extension.